bokeh/sphinxext/bokeh_roles.py

```diff
@@ -129,9 +159,26 @@ def bokeh_pull(name, rawtext, text, lineno, inliner, options=None, content=None)
     return [node], []
 
 
+def bokeh_requires(name, rawtext, text, lineno, inliner, options=None, content=None):
+    """Provide the minimum supported Python version from setup.py.
```

Comment:
is it possible this should be:

```suggestion
    """Provide the minimum required Python version from setup.py.
```
?

Reply:
Either/Or "supported"  connotes more of a statement about what promises we are making and what is maintained under CI